spec/cocina/generator/schema_array_spec.rb

```diff
@@ -5,14 +5,14 @@
 RSpec.describe Cocina::Generator::SchemaArray do
   # This tests the outcome of running exe/generator generate against openapi.yml.
 
-  context 'when an array of datatypes' do
-    # DescriptiveBasicValue.standard is an array of strings
-    let(:value) { Cocina::Models::DescriptiveBasicValue.new(standard: %w[marc bibframe]) }
-
-    it 'maps datatypes' do
-      expect(value.standard).to eq(%w[marc bibframe])
-    end
-  end
+  # context 'when an array of datatypes' do
+  #   # DescriptiveBasicValue.standard is an array of strings
+  #   let(:value) { Cocina::Models::DescriptiveBasicValue.new(standard: %w[marc bibframe]) }
+  #
+  #   it 'maps datatypes' do
+  #     expect(value.standard).to eq(%w[marc bibframe])
+  #   end
+  # end
```

Comment:
this is commented out ... should it be replaced with something else?

Reply:
It should probably be deleted -- there are no longer any string arrays in the Description model.